lib/pbench/test/unit/server/test_requests.py

```diff
@@ -90,13 +91,16 @@ def test_bad_extension_upload(client, bad_extension):
 
     @staticmethod
     def test_upload(client, pytestconfig):
-        filename = "./lib/pbench/test/unit/server/fixtures/upload/log.tar.xz"
+        filename = "log.tar.xz"
+        datafile = Path("./lib/pbench/test/unit/server/fixtures/upload/", filename)
```

Comment:
Do we have to have this hardwired?

Reply:
This is just for the one unit test, we dont store it anywhere in config file at the moment